### PR TITLE
Swiper & Styling

### PR DESCRIPTION
**Pull request by oldchevy**
npm install to get swiper

### CODE REVIEW

**Comment by p-herbert** [comment]
I do not see the debounced filterTag functions

File: client/journalapp/app/Main.js [hunk 1]
```diff
@@ -316,7 +329,39 @@ export default class Main extends Component {
...
+          <Swiper style={styles.wrapper} 
+                  showsButtons={true}
+                  loop={false}
+                  onMomentumScrollEnd={(e, state, context)=> {this.setState({page: this.state.slideView[state.index]});}}>
+           <View style={styles.slide1}>
+             <EntriesTab
+               navigator={navigator}
+               getEntries={ this.getEntries.bind(this) }
+               entries={ this.state.entries }
+               filterTags= { _.debounce(this.filterTags.bind(this), 500) }/>
```
Comment (oldchevy) [inline client/journalapp/app/Main.js:348]: One debounce

File: client/journalapp/app/Main.js [hunk 2]
```diff
@@ -316,7 +329,39 @@ export default class Main extends Component {
...
+           </View>
+           <View style={styles.slide2}>
+             <FeedTab
+             navigator={ navigator }
+             filterTags= { _.debounce(this.filterTags.bind(this), 500) }/>
```
Comment (oldchevy) [inline client/journalapp/app/Main.js:353]: The other debounce

**Comment by oldchevy** [comment]
I commented on where they are

**Pull request merged**